demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -1169,3 +1197,39 @@ def get_id_set_file(skip_dependencies, id_set_path):
         else:
             id_set = open_id_set_file(id_set_path)
         return id_set
+
+    @staticmethod
+    def check_and_validate_deprecated(file_path, current_file, is_modified, is_backward_check, validator):
+        """If file is deprecated, validate it. Return None otherwise.
+
+        Files with 'deprecated: true' or 'toversion < 4.5.0' fields are considered deprecated.
+
+        Args:
+            file_path: (str) file path to validate.
+            current_file: (dict) file in json format to validate.
+            is_modified: (boolean) for whether the file was modified.
+            is_backward_check: (boolean) for whether to preform backwards compatibility validation.
+            validator: (ContentEntityValidator) validator object to run backwards compatibility validation from.
+
+        Returns:
+            True if current_file is deprecated and valid.
+            False if current_file is deprecated and invalid.
+            None if current_file is not deprecated.
+        """
+        is_deprecated = "deprecated" in current_file and current_file["deprecated"]
+        toversion_is_old = "toversion" in current_file and \
+                           version.parse(current_file["toversion"]) < version.parse("4.5.0")
```

Comment:
Don't use the hardcoded 4.5.0 - We have a constant to indicate this: `OLDEST_SUPPORTED_VERSION`.
That way in the future we simply edit the constant and the same check will apply.